en/docs/assets/attachments/InstallAndSetup/apim200_to_apim260_gateway_artifact_migrator.ps1

```diff
@@ -0,0 +1,26 @@
+param (
```

Comment:
Shall we rename this file name to "apim200_to_apim300_gateway_artifact_migrator.ps1"

Reply:
Fixed